js/hotgridPopupView.js

```diff
@@ -0,0 +1,138 @@
+define([
+    'core/js/adapt'
+], function(Adapt) {
+    'use strict';
+
+    var HotgridPopupView = Backbone.View.extend({
+
+        className: 'hotgrid-popup',
+
+        events: {
+            'click .hotgrid-popup-done': 'closePopup',
+            'click .hotgrid-popup-controls': 'onControlClick'
+        },
+
+        initialize: function() {
+            this.listenToOnce(Adapt, "notify:opened", this.onOpened);
```

Comment:
Sorry it's picky I know but could you put single quotes around `notify:opened` for consistency?